examples/example-controllers/src/gas-prices-controller.ts

```diff
@@ -121,7 +121,7 @@ type AllowedEvents = never;
  * The messenger which is restricted to actions and events accessed by
  * {@link GasPricesController}.
  */
-export type GasPricesControllerMessenger = RestrictedControllerMessenger<
+export type GasPricesMessenger = RestrictedMessenger<
```

Comment:
Hmm, should we still call controller messengers `<Name>ControllerMessenger`? I know it is a mouthful to say, but it aligns with how we are naming other controller types (`GasPricesControllerState`, `GasPricesControllerActions`, etc.)

Reply:
Good question. We have examples of both conventions (e.g. `NetworkState`, `PreferencesState`).

I can update this to leave the names as they were before for now at least.

Reply:
Done here: 82537f28da051a11b699f85e09301a7f9e66395c